README.md

```diff
@@ -45,7 +45,7 @@ bundle install
 
 from this directory.
 
-Usage
+Usagedwad
```

Comment:
Thank you for your meaningful and well-researched contribution to the open source community.